Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -260,25 +209,30 @@ def main():
             'HumanReadableFormat': formats['markdown'],
         }
         demisto.results(entry)
-
-    train_tag_data = map(lambda x: x[DBOT_TAG_FIELD], data)
-    train_text_data = map(lambda x: x[text_field], data)
-    if len(train_text_data) != len(train_tag_data):
-        return_error("Error: data and tag data are different length")
-
     # print important words for each category
     try:
         find_keywords(data, DBOT_TAG_FIELD, text_field, keyword_min_score)
```

Comment:
Another small change - add boolean argument to the script, whether to extract keywords or not